packages/client/src/components/app/blocks/RowExplorer.svelte

```diff
@@ -3,25 +3,35 @@
   import BlockComponent from "components/BlockComponent.svelte"
   import { makePropSafe as safe } from "@budibase/string-templates"
   import { generate } from "shortid"
+  import { get } from "svelte/store"
+  import { getContext } from "svelte"
 
   export let dataSource
   export let height
-
   export let cardTitle
   export let cardSubtitle
   export let cardDescription
   export let cardImageURL
   export let cardSearchField
-
   export let detailFields
   export let detailTitle
-
   export let noRowsMessage
 
   const stateKey = generate()
+  const context = getContext("context")
+  const { generateGoldenSample } = getContext("sdk")
 
   let listDataProviderId
   let listRepeaterId
+
+  // Provide additional data context for live binding eval
+  export const getAdditionalDataContext = () => {
+    const rows = get(context)[listDataProviderId]?.rows || []
+    const goldenRow = generateGoldenSample(rows)
```

Comment:
What is the "Golden Sample" in this context?

Reply:
This was a utility I added to try and provide the "best" example of a row we can find. It's commented a bit at the implementation of `generateGoldenSample` but it will basically loop through all rows and take values for every column that isn't nullish.

It basically just accounts for the fact that if we just used the first row for live eval, there's a chance that the row you pick is mostly empty, so can't provide live eval for some fields.

This alleviates that by searching all rows to find a value for every column, where possible. It also prefers longer values too for things like arrays - basically just tries to give you a nice example for every column.